data_collection/web/guides.rst

```diff
@@ -9,4 +9,247 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
 Downloads and Outlinks
 ----------------------
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop p
+Downloads
+^^^^^^^^^
+
+Usually, you want to track when visitors download files from your website, a document, a file, a photo. Piwik PRO will automatically track clicks on such links as Downloads, and reports them in `Downloads` report.
```

Comment:
```suggestion
Download data helps you learn which files people pick from your site — be it a white paper, a case study, or a guide in pdf. Piwik PRO will automatically track clicks on such links as `Downloads`, and reports them in `Downloads` report.
```
I'd change the intro into sth like the txt used also in our HC

Reply:
changed